x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/infra_logs/kpi_over_time_config.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { ConfigProps, SeriesConfig } from '../../types';
+import { FieldLabels, RECORDS_FIELD, ReportTypes } from '../constants';
+import { LOG_RATE as LOG_RATE_FIELD } from '../constants/field_names/infra_logs';
+import { LOG_RATE as LOG_RATE_LABEL } from '../constants/labels';
+
+export function getLogsKPIConfig(configProps: ConfigProps): SeriesConfig {
+  return {
+    reportType: ReportTypes.KPI,
+    defaultSeriesType: 'bar',
+    seriesTypes: [],
+    xAxisColumn: {
+      label: i18n.translate('xpack.observability.exploratoryView.logs.logRateXAxisLabel', {
+        defaultMessage: 'Timestamp',
+      }),
+      dataType: 'date',
+      operationType: 'date_histogram',
+      sourceField: '@timestamp',
```

Comment:
Yeah, so to unravel this one would require the configuration generation function to be passed our real `DataView`. This would be a similar change to making sure our `DataView` is used, and a secondary one with the funky patterns isn't created and persisted. It would all be part of the same changeset (if it were to happen).